html5/shared/utils.js

```diff
@@ -76,6 +76,11 @@ function hasOwn (obj, key) {
  */
 
 function bind (fn, ctx) {
+  if (typeof fn !== 'function') {
```

Comment:
Well, the function `bind` is a shared util method. I suggestion keep it simple and quick. If you want to ignore errors when function binding it's better do the check in some particular situation code (for example event binding).
Thanks.